packages/components/src/components/colors.ts

```diff
@@ -136,9 +182,9 @@ export const colors = {
   ronchi: colorDictionary.ronchi,
   yellow500: colorDictionary.yellow500,
   blue500: colorDictionary.blueCrystal,
-  grey300: colorDictionary.grey300,
-  grey600: colorDictionary.grey600,
-  grey800: colorDictionary.grey800,
+  // grey300: colorDictionary.grey300,
+  // grey600: colorDictionary.grey600,
+  // grey800: colorDictionary.grey800,
```

Comment:
@jpye-finch if these lines can be commented out, can you delete them if no longer required please?